Justfile

```diff
@@ -149,6 +149,10 @@ test-integration *ARGS: devenv build-databuilder
 test-integration-no-spark *ARGS: devenv build-databuilder
     $BIN/python -m pytest -m "integration and not spark" {{ ARGS }}
 
+# run the run-databuilder-in-docker tests only
+test-in-docker *ARGS: devenv build-databuilder
```

Comment:
Maybe `test-docker` would be more consistent with our other targets? I don't feel v. strongly.

Reply:
My only reasoning for the `in-` was to avoid ambiguity around what exactly the `docker` part refers to since we have databases in docker, build the tool in docker, etc.  I don't feel strongly about it either though.

Reply:
I think consistency is probably more important here. We're going to have five kinds of tests (I think, currently): spec, unit, integration, docker and acceptance, so I think we should have matching targets for the three of them. There's a separate need for developers to understand what each of those kinds _are_, but that will need documentation (which is planned elsewhere).